client/packages/system/src/Immunisation/ImmunisationProgramDetailView/AppBarButtons.tsx

```diff
@@ -17,9 +17,9 @@ export const AppBarButtons = ({}: ProgramAppBarButtonsProps) => {
       <Grid container gap={1}>
         <ButtonWithIcon
           Icon={<PlusCircleIcon />}
-          label={t('button.add-new-immunisation')}
+          label={t('button.add-new-vaccine-course')}
```

Comment:
Looks like you moved this translation to `catalogue.json` but didn't update the useTranslation call here?

![Screenshot 2024-06-11 at 12 34 27 PM](https://github.com/msupply-foundation/open-msupply/assets/55115239/97e23a44-f4de-4f43-9e4b-a51dbe6fbab1)

Reply:
Good catch!